os/fs/smartfs/smartfs.h

```diff
@@ -258,6 +260,9 @@ struct smartfs_entry_s {
 	uint16_t dsector;			/* Sector number of the directory entry */
 	uint16_t doffset;			/* Offset of the directory entry */
 	uint16_t dfirst;			/* 1st sector number of the directory entry */
+#ifdef CONFIG_SMARTFS_USE_SECTOR_BUFFER
+	uint16_t last_psector;		/* Last sector in old parent directory chain, will be used to update chain after writing entry */
```

Comment:
How about below?
```
uint16_t prev_'d'sector;     /* Directory entry which will be chained to current dsector at the last time */
```

Reply:
Minor change, can do this before merging once logic is complete.